packages/synapse-react-client/src/components/TanStackTable/StyledTanStackTable.tsx

```diff
@@ -48,24 +88,7 @@ export default function StyledTanStackTable<T = unknown>(
             )
           })}
         </thead>
-        <tbody>
-          {table.getRowModel().rows.map(row => (
-            <tr key={row.id}>
-              {row.getVisibleCells().map(cell => {
-                return (
-                  <td
-                    key={cell.id}
-                    style={{
-                      width: cell.column.getSize(),
-                    }}
-                  >
-                    {flexRender(cell.column.columnDef.cell, cell.getContext())}
-                  </td>
-                )
-              })}
-            </tr>
-          ))}
-        </tbody>
+        <TableBody table={table} cellRenderer={cellRenderer} />
```

Comment:
Extract the table body so we can memoize it, which is required to take advantage of the column resizing optimization